vertx-service-discovery-bridge-consul/src/main/java/io/vertx/servicediscovery/consul/ConsulServiceImporter.java

```diff
@@ -123,7 +120,7 @@ private void retrieveIndividualServices(ServiceList list, Future<List<ImportedCo
     list.getList().forEach(service -> {
 
       Future<List<ImportedConsulService>> future = Future.future();
-      client.catalogServiceNodes(service.getName(), ar -> {
+      client.healthServiceNodes(service.getName(),true, ar -> {
```

Comment:
It is very useful to discovery not only the services in `passing` health state (for example, passing & warning). I suggest to set the `passing` argument to `false`, add level of discovery to the configuration and add additional filtering to `importService()` method

Reply:
I can see how this may be useful - have made the required changes.